src/Components/OATPropertyEditor/PropertySelector.tsx

```diff
@@ -161,43 +161,41 @@ const PropertySelector = ({
         >
             <Stack horizontal>
                 <div className={propertyInspectorStyles.propertyTagsWrapSecond}>
-                    {propertySelectorData.propertyTags.sectionSecond.map(
-                        (tag, i) => {
-                            if (
-                                lastPropertyFocused &&
-                                typeof lastPropertyFocused.item.schema ===
-                                    'object' &&
-                                tag.complex
-                            ) {
-                                return <></>;
-                            } else {
-                                return (
-                                    <ActionButton
-                                        onClick={() => {
-                                            onTagClick(tag.name);
-                                        }}
-                                        onKeyPress={() => {
-                                            onTagClick(tag.name);
-                                        }}
-                                    >
-                                        <Svg
-                                            tabIndex={0}
-                                            key={i}
-                                            className={
-                                                propertyInspectorStyles.propertyTag
-                                            }
-                                            src={tag.icon}
-                                            title={t(tag.title)}
-                                        ></Svg>
-                                    </ActionButton>
-                                );
-                            }
+                    {propertySelectorData.propertyTags.complex.map((tag, i) => {
+                        if (
+                            lastPropertyFocused &&
+                            typeof lastPropertyFocused.item.schema ===
+                                'object' &&
+                            tag.complex
+                        ) {
+                            return <></>;
+                        } else {
+                            return (
+                                <ActionButton
+                                    onClick={() => {
+                                        onTagClick(tag.name);
+                                    }}
+                                    onKeyPress={() => {
+                                        onTagClick(tag.name);
+                                    }}
+                                >
+                                    <Svg
+                                        tabIndex={0}
```

Comment:
Should we be hardcoding a zero tab index here? Or should we be using `i`? Not sure tbh.